### DISCUSSION TITLE
Proxy Flag help

### DISCUSSION THREAD

**Discussion by ayoubackup** [projectdiscovery/httpx#902] [General] [answered]
how i can use proxy flag with username and password in httpx or nuclei tools
example:  cat hosts.txt |  httpx  -sc -http-proxy http://152.44.97.146:8000  BUT i have username and password where i can put them

**Comment by forgedhallpass** [comment, marked as answer]
In general it should look something like this: `http[s]/socks5://[username:password@]host:port`

e.g. `https://myuser:mypassword@myhost:8080`
